### PR TITLE
Update Flowcell trends and Read count totals to bootstrap 5

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

**Review by sofiahaglund — COMMENTED** [review]
Wow, love how the flowcells trends plot looks now! I like the boxes at the bottom :-)

Everything seems to work except the datepicker in the FC trends plot? I tried to look at another time interval and the plot wasn't loading. Hmm I can try to find out why, could be my browser again...

**Review by sofiahaglund — APPROVED** [review]

File: run_dir/design/flowcell_trend_plot.html [hunk 1]
```diff
@@ -15,45 +15,53 @@ <h1>Flowcells Trend Plot</h1>
...
+              <button id="submit_interval" class="btn btn-primary btn-sm mt-3">Search</button>
+             </div>
             </div>
-            <button id="submit_interval" class="btn btn-primary btn-sm">Search</button>
-            </form>
+          </form>
         </div>
-        <div class="col-md-4">
+        <div class="col-md-5 d-flex justify-content-center">
+          <div class="d-flex justify-content-center border border-secondary p-3 rounded-3" style="box-sizing: border-box; -moz-box-sizing: border-box; width: 50%">
```
Comment (alneberg) [inline run_dir/design/flowcell_trend_plot.html:39]: Don't like to have the 'style' attribute at all. The width should be possible with the `w-50` class right?

File: run_dir/design/flowcell_trend_plot.html [hunk 2]
```diff
@@ -15,45 +15,53 @@ <h1>Flowcells Trend Plot</h1>
...
             <div id="display_type">
                 <form class='form-inline' id="display_type_form">
```
Comment (alneberg) [inline run_dir/design/flowcell_trend_plot.html:41]: Should probably remove the `form-inline` class here and elsewhere

File: run_dir/design/flowcell_trend_plot.html [hunk 3]
```diff
@@ -15,45 +15,53 @@ <h1>Flowcells Trend Plot</h1>
 <div id="main_plot" style="height: 700px;">
 </div>
 <div id="control_container" style="margin-top:40px;">
```
Comment (alneberg) [inline run_dir/design/flowcell_trend_plot.html:17]: Maybe have the style transferred to a margin class?

File: run_dir/design/flowcell_trend_plot.html (hunk 1 above)
Comment (alneberg) [reply run_dir/design/flowcell_trend_plot.html:39]: But feel free to keep it, its in so many other places on this page, so might not make much of a difference. It is entirely possible that it's I that need to change as well.

File: run_dir/design/flowcell_trend_plot.html (hunk 2 above)
Comment (aanil) [reply run_dir/design/flowcell_trend_plot.html]: Done!

File: run_dir/design/flowcell_trend_plot.html (hunk 1 above)
Comment (aanil) [reply run_dir/design/flowcell_trend_plot.html]: Got rid of some of it.

**Review by alneberg — APPROVED** [review]
Yes! Let's go! I am not a huge fan of the boxes. But the alignment is a huge improvement over what we had before.

**Pull request merged**